pkg/wrangler/context.go

```diff
@@ -456,6 +456,9 @@ func NewContext(ctx context.Context, clientConfig clientcmd.ClientConfig, restCo
 		core:         core,
 		api:          api,
 		crd:          crd,
+		capi:         capi,
+		rke:          rke,
+		rbac:         rbac,
```

Comment:
what's the point of adding these ?

Reply:
It's a fix, I missed them [here](https://github.com/rancher/rancher/pull/39876/files#diff-a386241816deb9f14c0dbac3c18f0642a47992060fe0f582782dd8be506d2c31R151-R164).